source/elements/oneDPL/source/parallel_api.rst

```diff
@@ -688,5 +688,35 @@ than an element in the range being searched.
 
 The elements e of [start, end) must be partitioned with respect to the comparator used.
 
+.. code:: cpp
+
+  template <class Policy, class InputIt, class OutputIt, class UnaryOp, class UnaryPredicate>
+  OutputIt
+  transform_if(Policy&& policy, InputIt start, InputIt end, OutputIt result, UnaryOp op,
+      UnaryPredicate pred);                                                               // (1)
+
+  template <class Policy, class InputIt1, class InputIt2, class OutputIt, class BinaryOp, 
+      class BinaryPredicate>
```

Comment:
Let's use `typename` for template type parameters, for consistency with other APIs on this page.

Reply:
done.